website/blog/2021-05-31-tungsten.md

```diff
@@ -0,0 +1,102 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.10.4 - Tungsten
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.10.4, which adds support for Scala 2.12.14. 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">24</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">11</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">4</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
```

Comment:
I would say the 2.12.14 is a new feature? That's what I want to believe anyway :sweat_smile: 

```suggestion
    <td align="center">1</td>
```